app/Http/Kernel.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Ushahidi\App\Http;
+
+use Illuminate\Foundation\Http\Kernel as HttpKernel;
+
+class Kernel extends HttpKernel
+{
+    /**
+     * The application's global HTTP middleware stack.
+     *
+     * These middleware are run during every request to your application.
+     *
+     * @var array
+     */
+    protected $middleware = [
+        \Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode::class,
+        \Illuminate\Foundation\Http\Middleware\ValidatePostSize::class,
+        \Ushahidi\App\Http\Middleware\TrimStrings::class,
+        // \Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class,
+        \Ushahidi\App\Http\Middleware\TrustProxies::class,
+        \Ushahidi\App\Http\Middleware\AddContentLength::class,
+        \Ushahidi\App\Multisite\DetectSiteMiddleware::class,
+        \Barryvdh\Cors\HandleCors::class,
+        \Ushahidi\App\Http\Middleware\MaintenanceMode::class,
+        \Ushahidi\App\Http\Middleware\SetLocale::class,
+        \v5\Http\Middleware\V5GlobalScopes::class,
+    ];
+
+    /**
+     * The application's route middleware groups.
+     *
+     * @var array
+     */
+    protected $middlewareGroups = [
+        'web' => [
+            // \Ushahidi\App\Http\Middleware\EncryptCookies::class,
+            // \Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse::class,
+            // \Illuminate\Session\Middleware\StartSession::class,
+            // \Illuminate\Session\Middleware\AuthenticateSession::class,
+            // \Illuminate\View\Middleware\ShareErrorsFromSession::class,
+            // \Ushahidi\App\Http\Middleware\VerifyCsrfToken::class,
+            // \Illuminate\Routing\Middleware\SubstituteBindings::class,
+        ],
+
+        'api' => [
+            'throttle:60,1',
```

Comment:
I have concerns about the value of the throttle "rate limit", and I think we need to raise it because this number is applied to all APIs together and not to each API alone @tuxpiper @webong

Reply:
For more information about the rate limit, it works on the public IP of the network, so if I exceed the rate limit while using my device, all other devices on the same private "wifi" network will also be blocked!

Reply:
Oh really, I used to think the underlying of the rate-limiting middleware applies the counts per route. 

So with this information, do we take out the throttle middleware from the route group or do we just increase the number of request counts.